rotkehlchen/db/utils.py

```diff
@@ -19,7 +19,8 @@
 
 DB_CREATE_USER_CREDENTIALS = """
 CREATE TABLE IF NOT EXISTS user_credentials (
-    name VARCHAR[24] NOT NULL PRIMARY KEY,
+    id INTEGER PRIMARY KEY,
```

Comment:
@LefterisJP Do you have an approach for DB upgrades in mind?

Reply:
Hmm so far it was only needed once. To upgrade the stored ETH accounts to a checksummed format. The commit is this: b31a360dd291ac0539bdf106bfd69817455a3025

The update happens here: https://github.com/rotkehlchenio/rotkehlchen/blob/2193dfe2215f59c97d3bbc809e5ec6edd5afcac5/rotkehlchen/db/dbhandler.py#L226-L238

Essentially if the version is 1 then it does the upgrade and it changes the version to 2. I suppose the same approach can be built upon with version 2->3 e.t.c.